packages/backend/src/Controllers/Election/getElectionResultsController.ts

```diff
@@ -36,7 +36,8 @@ const getElectionResults = async (req: IElectionRequest, res: Response, next: Ne
         const cvr: number[][] = []
         ballots.forEach((ballot: Ballot) => {
             const vote = ballot.votes.find((vote) => vote.race_id === race_id)
-            if (vote) {
+            let maxScoreGiven = vote?.scores.reduce((prev, s) => Math.max(prev, s.score ?? 0), 0) ?? 0;
```

Comment:
This logic should be done in the tabulation parser. 
Currently it looks like we're marking undervotes as not valid but still pushing them to the scores that are used.
```
else if (ballotValidity.isUnderVote) {
            underVotes += 1
            scores.push(row)  <-----
            // under votes should not count as a valid vote
            //validVotes.push(voter);
        }
```

Reply:
Yeah that's better. Fixed!